src/JSInterop/Microsoft.JSInterop/src/Infrastructure/DotNetDispatcher.cs

```diff
@@ -222,6 +222,8 @@ private static void EndInvokeDotNetAfterTask(Task task, JSRuntime jsRuntime, in
                 throw new JsonException($"Unexpected JSON token {reader.TokenType}. Ensure that the call to `{methodIdentifier}' is supplied with exactly '{parameterTypes.Length}' parameters.");
             }
 
+            jsRuntime.ResetByteArraysToBeRevived();
```

Comment:
Does the byte array leak if we encounter any of the errors thrown before this?

Reply:
No harm in moving this up before these two checks (to line 213). Even if this remains uncleared, it'll get cleared on the next message that supplies byte arrays.